### PR TITLE
Load the language data once into entities 

### PR DESCRIPTION
**Pull request by jcoyne**
Previously it was loaded every time the language popup was activated.

### CODE REVIEW

File: src/actionCreators/languages.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    return // Languages already loaded
+  }
+
+  dispatch(loadingLanguages())
+  fetch('https://id.loc.gov/vocabulary/iso639-1.json')
+    .then(resp => resp.json())
+    .then((json) => {
+      dispatch(languagesReceived(json))
+    })
+    .catch(() => false)
```
Comment (justinlittman) [inline src/actionCreators/languages.js]: Log the error?

File: src/components/editor/property/InputLang.jsx [hunk 2]
```diff
@@ -29,66 +24,19 @@ class InputLang extends Component {
...
-                    isLoading: false,
-                    options: this.createOptions(json),
-                  })
-                })
-                .catch(() => false)
-            }}
-            onBlur={() => {
-              this.setState({ isLoading: false })
-            }}
+            onFocus={() => this.props.loadLanguages()}
```
Comment (justinlittman) [inline src/components/editor/property/InputLang.jsx]: How about loading in App.componentDidMount()? That way it is (1) preloaded and (2) available for rendering language button when loading a resource.

File: src/actionCreators/languages.js (hunk 1 above)
Comment (jcoyne) [reply src/actionCreators/languages.js]: This is just moving this code here. This is exactly how it was written when it was in https://github.com/LD4P/sinopia_editor/pull/1012/files#diff-2435c181d99c6352e3e1322a7ce3eb43L85

File: src/components/editor/property/InputLang.jsx (hunk 2 above)
Comment (jcoyne) [reply src/components/editor/property/InputLang.jsx]: That's fine as a subsequent change, but this is how it was done previously.

**Pull request merged**